stdlib/src/base64/_b64encode.mojo

```diff
@@ -205,7 +205,7 @@ fn store_incomplete_simd[
     ]()
 
     memcpy(dest=pointer, src=tmp_buffer_pointer, count=nb_of_elements_to_store)
-    _ = simd_vector  # We make it live long enough
+    del simd_vector^  # We make it live long enough
```

Comment:
I don't htink this is necessary, have you tried just removing the `_ = simd_vector`?  I fixed lifetime extension of unsafe pointers.